### PR TITLE
Set project + Add eurostat_get functions (tsv)

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+def readlocal(fname):
+    return open(os.path.join(os.path.dirname(__file__), fname)).read()
+
+
+DISTNAME = 'larray_eurostat'
+VERSION = '0.1'
+AUTHOR = ' Alix Damman, Gaetan de Menten, Geert Bryon, Johan Duyck'
+AUTHOR_EMAIL = 'ald@plan.be'
+DESCRIPTION = "Additional package to import Eurostat files using LArray"
+LONG_DESCRIPTION = readlocal("README.rst")
```
Comment (gdementen) [inline setup.py]: does README.rst exist?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+from setuptools import setup, find_packages
+
+
+def readlocal(fname):
+    return open(os.path.join(os.path.dirname(__file__), fname)).read()
+
+
+DISTNAME = 'larray_eurostat'
+VERSION = '0.1'
+AUTHOR = ' Alix Damman, Gaetan de Menten, Geert Bryon, Johan Duyck'
```
Comment (gdementen) [inline setup.py]: extra leading whitespace

File: larray_eurostat/tsv.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    Returns
+    -------
+    LArray
+        Array containing indicator.
+    '''
+    tsv = indicator + ".tsv"
+    gz = tsv + ".gz"
+    url = _eurostat_path + gz
+    file_name, headers = urllib.request.urlretrieve(url, path + gz)
```
Comment (gdementen) [inline larray_eurostat/tsv.py]: Ideally, we should do everything in memory with something like:

```python
with urllib.request.urlopen(url) as f:
     with gzip.GzipFile(fileobj=f) as fgz:
         return la.read_eurostat(_cleantst(fgz.read()))
```
However I do not think la.read_eurostat supports a string :( but maybe using la.from_string would work?

File: larray_eurostat/tests/test_tsv.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+from unittest import TestCase
+import unittest
+from larray_eurostat.tsv import *
+
+
+class TestEurostat(TestCase):
+    def test_eurostat_get(self):
+        gdp = eurostat_get('nama_10_gdp')
+        self.assertEqual(gdp.dtype, float)
+        self.assertEqual(gdp.size, 1549548)
```
Comment (gdementen) [inline larray_eurostat/tests/test_tsv.py]: It would be nice to find a smaller test on eurostat website.

**Comment by gdementen** [comment]
maybe using `conda config --add channels gdementen` in before_install would help?

**Comment by alixdamman** [comment]
Travis fails simply because there is no larray package for linux environment (only for windows)

**Comment by gdementen** [comment]
haha! Oops... did not think about this. This will be resolved when larray is built via conda-forge. Until then I guess we will have to either disable travis for this project or install larray from source. Not sure it's worth the effort to install from source.

**Review by gdementen — APPROVED** [review]

**Comment by alixdamman** [comment]
Thanks for the review.

**Comment by alixdamman** [comment]
and for the rewriting of eurostat_get

**Pull request merged**